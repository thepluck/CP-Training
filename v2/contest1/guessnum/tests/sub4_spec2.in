4
129191
197723
204741
354348
