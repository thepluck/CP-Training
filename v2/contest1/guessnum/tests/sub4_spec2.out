3423796382
8019726211
8599166621
25757745388
